include/language.php

```diff
@@ -649,6 +649,17 @@ public function get_object_vars() {
 		return get_object_vars( $this );
 	}
 
+	/**
+	 * Converts current `PLL_language` into a stdClass object. Mostly used to allow dynamic properties.
+	 *
+	 * @since 3.4
+	 *
+	 * @return stdClass Converted `PLL_Language` object.
+	 */
+	public function to_standard_class() {
```

Comment:
Suggesting `to_std_class()`.
Also, for consistency, we could rename `get_object_vars()` to `to_array()`. What do you think ? In this case, this would require to rebase the branch before doing so.

Reply:
Good idea! Done in bc81d22.